nixos/modules/services/desktops/seatd.nix

```diff
@@ -0,0 +1,48 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
I think there is an ongoing effort to remove file-wide `with lib;` statements. Please use `inherit (lib) ...` or prefix with `lib.` (or a hybrid combination) instead.